nextflow.config

```diff
@@ -120,7 +120,6 @@ params {
     // Single cell raw input options
     // -----------------------
     reference_10x = null
-    sc_raw = false
```

Comment:
Not needed if setting the param `library_generation_protocol == "sc_10X_genomics"`

Reply:
sure, we can leave it out and add `library_generation_protocols` for the mixcr single cell analysis as well